easybuild/easyconfigs/c/CHARMPP/CHARMPP-8.0.0-foss-2023a.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'CHARMPP'
+version = '8.0.0'
+homepage = 'https://charmplusplus.org/tutorial/'
+
+description = """Charm++ is a message-passing parallel language and runtime system. 
+It is implemented as a set of libraries for C++. 
+It is efficient, and is portable to a wide variety of parallel machines. 
+Source code is provided, and non-commercial use is free."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+toolchainopts = {'usempi': True, 'openmp': False, 'pic': True}
+
+source_urls = ['http://charm.cs.illinois.edu/distrib/']
+sources = ['charm-%(version)s.tar.gz']
+checksums = ['e30fc1e921e5cbf3406e792d5b0ca5f211c5d8ffbfc56e56d5501d8118abcaf6']
+
+dependencies = [
+    ('CMake', '3.26.3'),
+    ('Tcl', '8.6.13'),
+    ('FFTW', '3.3.10'),
+]
+
+configure_without_installdir = "TRUE"
+
+configure_cmd = ' '
+
+build_cmd = ' export MPICXX=mpicxx ; ./build charm++ mpi-linux-x86_64 mpicxx --with-production --force '
```

no need to set `$MPICXX`:
```suggestion
build_cmd = "./build charm++ mpi-linux-x86_64 mpicxx --with-production --force "